src/chemnlp/data_val/model.py

```diff
@@ -43,17 +47,80 @@ class ColumnTypes(YamlStrEnum):
     continuous = "continuous"
     categorical = "categorical"
     ordinal = "ordinal"
+    boolean = "boolean"
 
 
 class Target(YamlModel):
     """Target information."""
 
     id: str
+
+    """A english description of the field"""
     description: str
-    units: str
+
+    """The units of the field. None if unitless."""
+    units: Optional[str]
+
+    """The type of the field. Can be one of `continuous`, `categorical`, `ordinal`, `boolean`."""
     type: ColumnTypes
+
+    """A list of names describing the field.
+
+    Note that this will be used in building the prompts. Some example for prompts:
+
+    - Boolean variables
+
+        - `Is <name> <identifier>?`
+        - ```
+        What molecules in the list are <name>?
+        - <identifier_1>
+        - <identifier_2>
+        - <identifier_3>
+        ```
+
+
+    - Continuous variables
+
+        - `What is <name> of <identifier>?`
+        - ```
+        What is the molecule with largest <name> in the following list?
+        - <identifier_1>
+        - <identifier_2>
+        - <identifier_3>
+        ```
+    """
     names: List[str]
 
+    """A URI or multiple (consitent ) URIs for the field.
+
+    Ideally this would be a link to an entry in an ontrology or controlled
+    vocabulary that can also provide a canonical description for the field.
+    """
+    uris: Optional[List[str]]
+
+    """A PubChem assay IDs or multiple (consistent) PubChem assay IDs.
+
+    Make sure that the first assay ID is the primary assay ID.
+    """
+    pubchem_aids: Optional[List[int]]
+
+    @validator("pubchem_aids")
+    def uris_resolves(cls, values):
+        if values is not None:
+            for uri in values.get("uris"):
```

Comment:
@kjappelbaum Because `pubchem_aids: Optional[List[int]]` I guess this needs to be something like `for uri in values:` and then we need to create the correct URL for the request?

Reply:
Because currently we get:
```
.../chemnlp/src/chemnlp/data_val/model.py", line 110, in uris_resolves
    for uri in values.get("uris"):
AttributeError: 'list' object has no attribute 'get'
```